docs/architecture/pk-4.md

```diff
@@ -320,18 +333,17 @@ mbedtls_pk_verify()
 mbedtls_pk_sign()
 ```
 
-ACTION: document that `mbedtls_pk_sign()` are legacy functions, that perform the same algorithm that `mbedtls_pk_get_psa_attributes()` would perform under the hood if given a sign or verify usage.
+ACTION (https://github.com/Mbed-TLS/TF-PSA-Crypto/issues/347): document that `mbedtls_pk_sign()` are legacy functions, that perform the same algorithm that `mbedtls_pk_get_psa_attributes()` would perform under the hood if given a sign or verify usage.
```

Comment:
I think this is missing a mention of `mbedtls_pk_verify()` (or else `s/are legacy functions/is a legacy function`)
```suggestion
ACTION (https://github.com/Mbed-TLS/TF-PSA-Crypto/issues/347): document that `mbedtls_pk_sign()` and `mbedtls_pk_verify()` are legacy functions, that perform the same algorithm that `mbedtls_pk_get_psa_attributes()` would perform under the hood if given a sign or verify usage.
```